sentry-kotlin-compiler-plugin/buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+    kotlin("jvm") version "1.8.20"
```

Comment:
`l`: maybe worth keeping it in sync with `plugin-build/buildSrc` (I mean the version). There it's `1.6.10`